test/services/copy_profiles_to_policies_test.rb

```diff
@@ -45,7 +45,7 @@ class CopyProfilesToPoliciesTest < ActiveSupport::TestCase
 
   test 'copies two similar internal profiles with to two new policies' do
     first = profiles(:one)
-    (bm = benchmarks(:one).dup).update!(version: '0.1.46')
+    (bm = benchmarks(:one).dup).update!(version: '0.1.46', ref_id: 'bm1')
```

Comment:
These two test updates were the only breakages where a test was creating two policies of the same policy type in the same account.